### DISCUSSION TITLE
Orpheus Sheet in the future?

### DISCUSSION THREAD

**Discussion by Beartoya** [JohanFalt/Foundry_WoD20#905] [Q&A] [answered]
Hello! Thank you for all your hard work on this module. I was wondering if theres any plans on making a Orpheus sheet for this module, I know there's rules for playing an Orpheus game with wraith but I want to use the rules how they are presented in the Orpheus run. Orpjeus isnt the most popular WoD game but I really like it,  and having a easy way to play it online would be great! Once again thank you for your time!

**Comment by JohanFalt** [comment, marked as answer]
I'll look into it for the coming version what can be done. Comments and progress can be found here:

https://github.com/JohanFalt/Foundry_WoD20/issues/907

**Comment by JohanFalt** [comment]
As I don't know Orpheus I went with what I do know. I'll added it as a Mortal variant and then handled it as every other WoD game there is. Their powers doesn't seem to have ratings or really rolls either so... well this was the endresult.

Release with v4.2.

Any comments or other things that I might have missed please add them here.

![{FBC2FDAD-C40E-4A6C-B893-652F5A1199C8}](https://github.com/user-attachments/assets/7db911aa-57e8-4e11-980b-622515e12451)

![{AE16A6DB-229B-454E-B8E8-AEDB1E4E635D}](https://github.com/user-attachments/assets/589bbd37-fb7e-46cc-8cb7-08d1975aa566)

![{7AD833F6-9005-431F-A277-F663E29D9305}](https://github.com/user-attachments/assets/8c276607-aca9-4def-9046-0d30078bca9a)
